model/src/main/java/com/ge/predix/acs/rest/AttributeConnector.java

```diff
@@ -32,6 +39,7 @@ public void setMaxCachedIntervalMinutes(final int maxCachedIntervalMinutes) {
         this.maxCachedIntervalMinutes = maxCachedIntervalMinutes;
     }
 
+    @ApiModelProperty(value = "A set of attribute store adapters associated with this connector", required = true)
```

Comment:
Might be more clearly worded as "A set of adapters used to retrieve attributes from"? Also should we mention here or in a controller-related Swagger annotation that only one adapter is currently supported?